config_data.yml

```diff
@@ -22,3 +22,13 @@ translations:
 # Extra fields to generate reporting stats for, for additional tabs
 # the "Reporting status" page.
 reporting_status_extra_fields: []
+
+# This builds one or more GeoJSON files to be used by Open SDG maps. For more
+# information on the parameters, see the sdg-build library.
```

Comment:
@brockfanning think once https://github.com/open-sdg/open-sdg/pull/774 has been merged, this could say `see the maps documentation: https://open-sdg.readthedocs.io/en/latest/maps/` instead.

Other than that, think this looks good.